larq_compute_engine/mlir/transforms/legalize_tflite.cc

```diff
@@ -0,0 +1,93 @@
+#include "flatbuffers/flexbuffers.h"
+#include "larq_compute_engine/mlir/ir/lce_ops.h"
+#include "mlir/IR/PatternMatch.h"
+#include "mlir/Pass/Pass.h"
+#include "tensorflow/compiler/mlir/lite/ir/tfl_ops.h"
+
+namespace mlir {
+namespace TFL {
+
+namespace {
+
+std::vector<uint8_t> buildCustomOption(TF::BsignOp op) { return {}; }
+
+std::vector<uint8_t> buildCustomOption(TF::Bconv2dOp op) {
+  flexbuffers::Builder fbb;
+  fbb.Map([&]() {
+    fbb.Int("channels_in", op.channels_in().getSExtValue());
+    fbb.Int("dilation_height_factor",
+            op.dilation_height_factor().getSExtValue());
+    fbb.Int("dilation_width_factor", op.dilation_width_factor().getSExtValue());
+    fbb.String("fused_activation_function",
+               std::string(op.fused_activation_function()));
+    fbb.Int("pad_values", op.pad_values().getSExtValue());
+    fbb.String("padding", std::string(op.padding()));
```

Comment:
In a future PR this will allow us to very easily use the TFLite enum here directly.